students/gangho/users/views.py

```diff
@@ -61,10 +64,13 @@ def post(self, request):
             validate_email(email)
             validate_password(password)
 
-            if not User.objects.filter(email=email, password=password).exists():
+            user_in_database = User.objects.get(email=email)
```

Comment:
`user`는 그냥 유저일 뿐입니다. 변수명에 상황을 설명할 필요는 없어보입니다!
```suggestion
            user = User.objects.get(email=email)
```

Reply:
`user = User.objects.get(email = email)` 에서 에러가 발생했을 경우에 except문에서 에러를 잡아줄 수 있도록 수정해주세요.

Reply:
user_in_database -> user 로 변수명 수정하였습니다. 
또한, user가 db에 없을 경우를 위해, User.DoesNotExist 로 예외처리 하였습니다.